src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -533,6 +533,9 @@ void PeakDetectorCLI::loadCompoundsFile() {
 		int loadCount = DB.loadCompoundCSVFile(mavenParameters->ligandDbFilename);
 		mavenParameters->compounds = DB.compoundsDB;
 		cout << "Total Compounds Loaded : " << loadCount << endl;
+		if (loadCount == 0) {
```

Comment:
It should be like this - 


if (loadCount == 0) {
			cerr << "Warning: Uploaded compound database is empty!" << endl;
		}
		else{
			cout << "Total Compounds Loaded : " << loadCount << endl;
		}

Reply:
@saifulbkhan While we're at it, also remove the word 'uploaded'. We're not uploading the data anywhere.